<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DiveLogExportDialog</class>
 <widget class="QDialog" name="DiveLogExportDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>507</width>
    <height>616</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Export Dive Log Files</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>560</y>
     <width>411</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>481</width>
     <height>491</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="General_tab">
    <attribute name="title">
     <string>General Export</string>
    </attribute>
    <widget class="QGroupBox" name="exportSelection">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>20</y>
       <width>171</width>
       <height>114</height>
      </rect>
     </property>
     <property name="title">
      <string>Selection</string>
     </property>
     <widget class="QRadioButton" name="exportSelected">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>151</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Selected dives</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QRadioButton" name="exportAll">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>110</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>All dives</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="exportFormat">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>20</y>
       <width>191</width>
       <height>221</height>
      </rect>
     </property>
     <property name="title">
      <string>Export format</string>
     </property>
     <widget class="QRadioButton" name="exportUDDF">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>110</width>
        <height>24</height>
       </rect>
      </property>
      <property name="maximumSize">
       <size>
        <width>110</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>UDDF</string>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">exportGroup</string>
      </attribute>
     </widget>
     <widget class="QRadioButton" name="exportDivelogs">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>100</y>
        <width>131</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>divelogs.de</string>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">exportGroup</string>
      </attribute>
     </widget>
     <widget class="QRadioButton" name="exportCSV">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>130</y>
        <width>110</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>CSV</string>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">exportGroup</string>
      </attribute>
     </widget>
     <widget class="QRadioButton" name="exportWorldMap">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>160</y>
        <width>171</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Worldmap</string>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">exportGroup</string>
      </attribute>
     </widget>
     <widget class="QRadioButton" name="exportSubsurfaceXML">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>171</width>
        <height>21</height>
       </rect>
      </property>
      <property name="maximumSize">
       <size>
        <width>171</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>Subsurface XML</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">exportGroup</string>
      </attribute>
     </widget>
    </widget>
    <widget class="QLabel" name="description">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>310</y>
       <width>341</width>
       <height>91</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="Line" name="line">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>250</y>
       <width>231</width>
       <height>16</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="Line" name="line_3">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>60</y>
       <width>20</width>
       <height>151</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="HTML_tab">
    <attribute name="title">
     <string>HTML</string>
    </attribute>
    <widget class="QGroupBox" name="advanceOptions">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>240</y>
       <width>441</width>
       <height>191</height>
      </rect>
     </property>
     <property name="title">
      <string>Style Options</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <widget class="QWidget" name="layoutWidget1">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>291</width>
        <height>29</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="fontLayout">
       <item>
        <widget class="QLabel" name="fontLabel">
         <property name="text">
          <string>Font</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="fontSelection"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>291</width>
        <height>29</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="fontSizeLayout">
       <item>
        <widget class="QLabel" name="fontSizeLabel">
         <property name="text">
          <string>Font size</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="fontSizeSelection">
         <property name="currentIndex">
          <number>3</number>
         </property>
         <item>
          <property name="text">
           <string>8</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>10</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>12</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>14</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>16</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>18</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>20</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>120</y>
        <width>291</width>
        <height>29</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="ThemeLayout">
       <item>
        <widget class="QLabel" name="themeLabel">
         <property name="text">
          <string>Theme</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="themeSelection">
         <property name="currentIndex">
          <number>0</number>
         </property>
         <item>
          <property name="text">
           <string>Light</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Sand</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QGroupBox" name="GeneralOptions">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>441</width>
       <height>201</height>
      </rect>
     </property>
     <property name="title">
      <string>General Settings</string>
     </property>
     <widget class="Line" name="line_2">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>40</y>
        <width>20</width>
        <height>141</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>191</width>
        <height>80</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="additionalOptions">
       <item>
        <widget class="QCheckBox" name="checkBox">
         <property name="text">
          <string>Order dives from 0</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox_2">
         <property name="text">
          <string>Minimum Javascript</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox_3">
         <property name="text">
          <string>Dive List only</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>50</y>
        <width>132</width>
        <height>52</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QRadioButton" name="exportSelectedDives">
         <property name="minimumSize">
          <size>
           <width>117</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>Selected Dives</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">buttonGroup</string>
         </attribute>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="exportAllDives">
         <property name="minimumSize">
          <size>
           <width>117</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>All Dives</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">buttonGroup</string>
         </attribute>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>20</y>
     <width>491</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Droid Sans [unknown]</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="text">
    <string>Export Dive Log Files</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DiveLogExportDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DiveLogExportDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <buttongroups>
  <buttongroup name="exportGroup"/>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
